angular-frontend/README.md

```diff
@@ -15,6 +15,7 @@ It provides ways to test the service and component logic.
 ## Prerequisites
 In order to further develop this application the following tools needs to be setup:
 - NodeJS LTS (https://nodejs.org/) with NPM
+- Angular CLI (https://angular.io/cli)
```

Comment:
The CLI comes as a dependency when npm install has been executed, so formally not a prerequisite. If installed globally it can differ from this version provided in the package.json.